homeassistant/components/elevenlabs/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
```

Comment:
Please add `data_description` where you give a bit more context about this

Reply:
I didn't even know this field existed, thanks for pointing this out!